### PR TITLE
Update how-to-write-a-parser.md to include advice for CVSS data

### PR DESCRIPTION
**Pull request by damiencarol**
* explain what to do
* talk about `cvss` module
* explain how we use `cvss` module

### CODE REVIEW

File: docs/content/en/contributing/how-to-write-a-parser.md [hunk 1]
```diff
@@ -103,6 +103,69 @@ Good example:
        finding.cve = data["mykey"]
 ```
 
+### Do not parser CVSS by hand (vector, score or severity)
```
Comment (valentijnscholten) [inline docs/content/en/contributing/how-to-write-a-parser.md:106]:
```suggestion
### Do not parse CVSS by hand (vector, score or severity)
```

File: docs/content/en/contributing/how-to-write-a-parser.md [hunk 2]
```diff
@@ -103,6 +103,69 @@ Good example:
...
+
+Data can have `CVSS` vectors or scores. Don't try to right again your own CVSS score algorythm.
```
Comment (valentijnscholten) [inline docs/content/en/contributing/how-to-write-a-parser.md:108]:
```suggestion
Data can have `CVSS` vectors or scores. Don't try to write your own CVSS score algorithm.
```

File: docs/content/en/contributing/how-to-write-a-parser.md [hunk 3]
```diff
@@ -103,6 +103,69 @@ Good example:
...
+For parser, we rely in module `cvss`.
```
Comment (valentijnscholten) [inline docs/content/en/contributing/how-to-write-a-parser.md:109]:
```suggestion
For parser, we rely on module `cvss`.
```

**Review by valentijnscholten — CHANGES_REQUESTED** [review]
made suggestions on typo's

**Comment by damiencarol** [comment]
@valentijnscholten committed your suggestions.

**Review by valentijnscholten — APPROVED** [review]

**Review by madchap — APPROVED** [review]

**Pull request merged**